assets/geomesa-quickstart-nifi/GeoMesa_Quickstart.xml

```diff
@@ -0,0 +1,374 @@
+<?xml version="1.0" ?>
+<template encoding-version="1.0">
```

Comment:
Good catch, I had a broken link in the docs below. This is the NiFi flow template. Once they have the processors installed they can upload this file and it wires up everything for them so they can just hit play.